changelog.d/7619.misc

```diff
@@ -0,0 +1 @@
+Check that called deferred methods succeed and general cleanup of `MonthlyActiveUsersTestCase` and `TestMauLimit`.
```

Comment:
"deferred methods" is confusing to me, we're checking that each `Deferred` has completed. I'm not sure it even matters that they are `Deferreds` though, so maybe:

```suggestion
Check that all asynchronous tasks succeed and general cleanup of `MonthlyActiveUsersTestCase` and `TestMauLimit`.
```